lib/takeaway.rb

```diff
@@ -0,0 +1,51 @@
+require_relative 'Menu'
+require("bundler")
+Bundler.require()
+
+
+class Takeaway
+  attr_reader :basket, :result,:menu
+
+  def initialize
+    @basket = []
+    @menu = nil
+    @total = 0
+  end
+
+  def see(menu)
+    @menu = menu.list
+  end
+
+  def select(item,quantity=1)
+    @result = @menu.find { |x| x[item] }
```

Comment:
Nice, this looks like it would work. I think you could simplify the method here to not need concat, by only pushing the item into the array.

Reply:
Hi Lindsey, Thank you for your suggestion. I have changed my code accordingly to use push (instead of concat) and it works :)